warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -9,10 +9,26 @@ models:
         tests: &primary_key_tests
           - not_null
           - unique
+        meta:
+          metabase.semantic_type: type/PK
   - name: dim_contracts
     description: '{{ doc("contracts_table") }}'
     columns:
       - *key
+      - name: contract_holder_organization_key
+        tests:
+          - relationships:
+             to: ref('dim_organizations')
+             field: key
+        meta:
+          metabase.semantic_type: type/FK
```

Comment:
I think dbt-metabase will already handle FKs based on relationships tests, so this is redundant

same in other places